src/AvaloniaEdit/Rendering/TextView.cs

```diff
@@ -1568,12 +1568,12 @@ public virtual void MakeVisible(Rect rectangle)
                 }
                 else
                 {
-                    newScrollOffsetY = rectangle.Y;
+                    newScrollOffsetY = rectangle.Y - 1; // -1 is temp fix
                 }
             }
             else if (rectangle.Bottom > visibleRectangle.Bottom)
             {
-                newScrollOffsetY = rectangle.Bottom - _scrollViewport.Height;
+                newScrollOffsetY = rectangle.Bottom - _scrollViewport.Height + 1; // +1 is temp fix
```

Comment:
Would you mind explaining what +1 would fix? same to line:1571

Reply:
That was not added by me, but looking at the source repo that "temp fix" was (coincidentally) removed 1 day after I opened this PR.